src/components/Article/Byline/Byline.styled.js

```diff
@@ -52,17 +52,19 @@ Container.propTypes = {
   children: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
 }
 
-export const Content = ({ children }) => 
+export const Content = ({ children, mb }) => 
   <Block
-    width='100%'
     align='column'
-    md={{ align: 'row', aligny: 'middle' }}
+    mb={mb[0]}
+    width='100%'
+    md={{ align: 'row', aligny: 'middle', mb: mb[1] }}
   >
     {children}
   </Block>
 
 Content.propTypes = {
-  children: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
+  children: PropTypes.oneOfType([PropTypes.array, PropTypes.object]),
+  mb: PropTypes.string
```

Comment:
Aqui tem que ser array
```suggestion
  mb: PropTypes.array
```